tests/tests_api/test_language_and_level.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from rest_framework import status
+
+from api.models import LanguageAndLevel
+
+
+@pytest.mark.parametrize("language", [None, "en", "fr"])
+def test_languages_and_levels_list(admin_client, language):
+    queryset = LanguageAndLevel.objects.all()
+
+    if language:
+        queryset = queryset.filter(language=language)
+        query_string = f"?language={language}"
+    else:
+        query_string = ""
+
+    response = admin_client.get(f"/api/languages_and_levels/{query_string}")
+    assert response.status_code == status.HTTP_200_OK
+    assert response.json() == [
+        {
+            "id": language_and_level.id,
+            "language": {
+                "id": language_and_level.language.id,
```

Comment:
actually this will be equal to `language` in test parametrization, not sure if you want to test this. Also, it can't be tested for language=`None`

Reply:
I think current solution makes it more understable. It's very clear which field is used in response, but parametrization is only about filtering.

>Also, it can't be tested for language=None

Didn't get it? Do yo mean `language_and_level` could exist without `language` (language=None)?

Reply:
Never mind, I was obviously not thinking clearly in the evening :)